parlemonde/clap
# features improvement

### Motivation

Fix bugs and add new features/improvements



### Changes

- https://trello.com/c/JEb3NZyf/22-feature-historique-des-feedbacks
- https://trello.com/c/zreWF0FU/47-feature-amplification-du-son
- https://trello.com/c/75Db8yWs/39-feature-am%C3%A9liorer-le-syst%C3%A8me-de-t%C3%A9l%C3%A9chargement
- https://trello.com/c/Hb08oMKT/48-feature-invisibiliser-le-bouton-de-retour
- https://trello.com/c/cdbWbOFm/44-bug-adaptation-du-texte-en-fonction-des-vignettes
- https://trello.com/c/kVee1iCU/40-feature-options-de-fond



### Test

- Display all feedback sent by the professor above feedback form
- Increase max sound volume 
- Remove modal before downloading video 
- Hide show feedbacks button on pre mounting step if no feedbacks 
- Title's style should be correctly applied in title card and diaporama component
- Try to change background color and text color of title
